app/package.json

```diff
@@ -22,76 +22,76 @@
+    "@jupyterlab/cell-toolbar-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/celltags": "~4.0.0-alpha.13",
+    "@jupyterlab/codeeditor": "~4.0.0-alpha.13",
+    "@jupyterlab/codemirror-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/collaboration": "~4.0.0-alpha.13",
+    "@jupyterlab/collaboration-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/completer": "~4.0.0-alpha.13",
+    "@jupyterlab/completer-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/console": "~4.0.0-alpha.13",
+    "@jupyterlab/console-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/coreutils": "~6.0.0-alpha.13",
+    "@jupyterlab/docmanager": "~4.0.0-alpha.13",
+    "@jupyterlab/docmanager-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/docprovider": "~4.0.0-alpha.13",
+    "@jupyterlab/docprovider-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/documentsearch": "~4.0.0-alpha.13",
+    "@jupyterlab/documentsearch-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/filebrowser": "~4.0.0-alpha.13",
+    "@jupyterlab/filebrowser-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/fileeditor": "~4.0.0-alpha.13",
+    "@jupyterlab/fileeditor-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/hub-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/javascript-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/json-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/mainmenu": "~4.0.0-alpha.13",
+    "@jupyterlab/mainmenu-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/markedparser-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/mathjax2-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/notebook": "~4.0.0-alpha.13",
+    "@jupyterlab/notebook-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/observables": "~5.0.0-alpha.13",
+    "@jupyterlab/outputarea": "~4.0.0-alpha.13",
+    "@jupyterlab/pdf-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/rendermime": "~4.0.0-alpha.13",
+    "@jupyterlab/rendermime-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/rendermime-interfaces": "~3.8.0-alpha.13",
+    "@jupyterlab/running-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/services": "~7.0.0-alpha.13",
+    "@jupyterlab/settingregistry": "~4.0.0-alpha.13",
+    "@jupyterlab/shared-models": "~4.0.0-alpha.13",
+    "@jupyterlab/shortcuts-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/statedb": "~4.0.0-alpha.13",
+    "@jupyterlab/statusbar": "~4.0.0-alpha.13",
+    "@jupyterlab/terminal": "~4.0.0-alpha.13",
+    "@jupyterlab/terminal-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/theme-dark-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/theme-light-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/tooltip": "~4.0.0-alpha.13",
+    "@jupyterlab/tooltip-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/translation": "~4.0.0-alpha.13",
+    "@jupyterlab/translation-extension": "~4.0.0-alpha.13",
+    "@jupyterlab/ui-components": "~4.0.0-alpha.28",
+    "@jupyterlab/vega5-extension": "~4.0.0-alpha.13",
+    "@lumino/algorithm": "^2.0.0-alpha.1",
+    "@lumino/application": "^2.0.0-alpha.1",
```

Comment:
Looks like we can also update the `@lumino` packages to `^2.0.0-alpha.6` which is the latest version at the moment.